testdata/fix/nginx-v1alpha1/Kptfile

```diff
@@ -0,0 +1,101 @@
+apiVersion: kpt.dev/v1alpha1
+kind: Kptfile
+metadata:
+  name: nginx
+packageMetadata:
+  shortDescription: describe this package
+  url: https://github.com/GoogleContainerTools/kpt
+  email: foo@gmail.com
+  license: license text
+  version: v0.1
+  tags:
+    - tag1
+    - tag2
+  man: nginx man text
+upstream:
+  type: git
+  git:
+    commit: 4d2aa98b45ddee4b5fa45fbca16f2ff887de9efb
+    repo: https://github.com/GoogleContainerTools/kpt
+    directory: package-examples/nginx
+    ref: v0.2
+dependencies:
+  - name: hello-world
+    git:
+      repo: https://github.com/GoogleContainerTools/kpt
+      directory: /package-examples/helloworld-set
+      ref: master
+    updateStrategy: fast-forward
+    ensureNotExists: false
+    autoSet: false
+    functions:
+      - image: gcr.io/kpt-fn/my-func:v0.1
+        config:
+          apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: function-config
+          data:
+            param1: value1
+    stdin:
+      filenamePattern: foo*
+      original: bar
+functions:
+  autoRunStarlark: true
+  starlarkFunctions:
+    - name: foo-star
+      path: path/to/foo-star.yaml
+openAPI:
+  definitions:
+    io.k8s.cli.setters.image:
+      x-k8s-cli:
+        setter:
+          name: image
+          value: nginx
+    io.k8s.cli.setters.namespace:
+      maxLength: 10
+      type: string
+      x-k8s-cli:
+        setter:
+          name: namespace
+          value: some-space
+    io.k8s.cli.setters.tag:
+      x-k8s-cli:
+        setter:
+          name: tag
+          value: 1.14.1
+    io.k8s.cli.substitutions.fullimage:
+      x-k8s-cli:
+        substitution:
+          name: fullimage
+          pattern: ${image}:${tag}
+          values:
+          - marker: ${image}
+            ref: '#/definitions/io.k8s.cli.setters.image'
+          - marker: ${tag}
+            ref: '#/definitions/io.k8s.cli.setters.tag'
+    io.k8s.cli.substitutions.imageidentifier:
+      x-k8s-cli:
+        substitution:
+          name: imageidentifier
+          pattern: deployment-${fullimage}
+          values:
+          - marker: ${fullimage}
+            ref: '#/definitions/io.k8s.cli.substitutions.fullimage'
+    io.k8s.cli.setters.list:
+      type: array
+      x-k8s-cli:
+        setter:
+          name: list
+          value: ""
+          listValues:
+            - dev
+            - stage
+inventory:
```

Comment:
what if the package is using `ConfigMap` inventory object?

Related to comment on documenting the limitation. Be very explicit, what it doesn't handle, and make sure it fails clearly instead of silently passing.

Reply:
I will work on results in next PR.